### PR TITLE
update README and index.js

### PR DESCRIPTION
**Pull request by MeoWu630**

### CODE REVIEW

**Review by Loverush — DISMISSED** [review]

File: basic/01-web3js-deploy/index.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ let Web3 = require('web3');
 let solc = require('solc');
 let fs = require('fs');
 
-// Get privatekey from sk.txt
+// Get privatekey from .env file
```
Comment (qizhou) [inline basic/01-web3js-deploy/index.js:5]:
```suggestion
// Get privatekey from environment
```

**Review by qizhou — DISMISSED** [review]

File: basic/03-web3js-erc20/README.md [hunk 2]
```diff
@@ -192,3 +192,7 @@ https://ethereum.stackexchange.com/questions/70853/the-method-eth-sendtransactio
 ## 参考文档  
 - Mocha 实例教程: http://www.ruanyifeng.com/blog/2015/12/a-mocha-tutorial-of-examples.html    
 - Mocha 技术笔记:  https://pcaaron.github.io/pages/fe/block/improve4.html#%E8%B7%91%E6%B5%8B%E8%AF%95   
+<<<<<<< HEAD
+=======
+- ERC20 接口及合约说明：https://docs.openzeppelin.com/contracts/2.x/api/token/erc20
```
Comment (liberhe) [inline basic/03-web3js-erc20/README.md:197]: please fix the conflict.

**Review by liberhe — CHANGES_REQUESTED** [review]

File: basic/03-web3js-erc20/README.md (hunk 2 above)
Comment (MeoWu630) [reply basic/03-web3js-erc20/README.md:195]: already fixed, sorry for that

**Review by liberhe — APPROVED** [review]

**Comment by liberhe** [comment]
please join our discord https://discord.com/invite/gkENMS7cKz and introduce yourself.  You can also add WeChat yanyanho666 to join our developer group.

**Review by yingjingyang — APPROVED** [review]

**Pull request merged**